src/core/path.c

```diff
@@ -555,7 +565,37 @@ static void path_enter_running(Path *p, char *trigger_path) {
         job_set_activation_details(job, details);
 
         path_set_state(p, PATH_RUNNING);
-        path_unwatch(p);
+
+        if (!p->deactivation_toggle)
+                path_unwatch(p);
+
+        return;
+
+fail:
+        path_enter_dead(p, PATH_FAILURE_RESOURCES);
+}
+
+static void path_deactivate_unit(Path *p) {
+        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+        Unit *trigger;
+        Job *job;
+        int r;
+
+        assert(p);
+
+        trigger = UNIT_TRIGGER(UNIT(p));
+        if (!trigger) {
+                log_unit_error(UNIT(p), "Unit to trigger vanished.");
+                goto fail;
+        }
+
+        r = manager_add_job(UNIT(p)->manager, JOB_STOP, trigger, JOB_REPLACE, NULL, &error, &job);
```

Comment:
you can specify NULL as last arg, given you don't actuall are about the job object here.

also, according to our newest coding style, please prefix arguments you pass 0/NULL to (i.e. where the parameter gives you no hint on what this argument is), with the argument name.  here: `/* affected_jobs= */ NULL`